src/pages/pool/_id.vue

```diff
@@ -1,17 +1,19 @@
 <template>
-  <div class="lg:container lg:mx-auto pt-8">
-    <div class="grid grid-cols-1 lg:grid-cols-3 gap-y-8 gap-x-0 lg:gap-x-8">
-      <div class="col-span-2">
+  <div class="xl:container xl:mx-auto lg:px-4 pt-8">
+    <div
+      class="grid grid-cols-1 lg:grid-cols-3 gap-y-8 gap-x-0 lg:gap-x-4 xl:gap-x-8"
+    >
+      <div class="col-span-2 px-4 lg:px-0">
         <BalLoadingBlock v-if="loadingPool" class="h-16" />
-        <div v-else class="px-4 lg:px-0 flex flex-col">
+        <div v-else class="flex flex-col">
           <div class="flex flex-wrap items-center -mt-2">
             <h3 class="pool-title">
               {{ poolTypeLabel }}
             </h3>
             <div
               v-for="([address, tokenMeta], i) in titleTokens"
               :key="i"
-              class="mt-2 mr-2 flex items-center px-2 h-10 bg-gray-50 dark:bg-gray-850 rounded-lg"
+              class="mt-2 mr-2 flex items-center h-10 bg-gray-50 dark:bg-gray-850 rounded-lg"
```

Comment:
is it really better to remove padding here?

Reply:
@alter-eggo Good catch - this was a mistake. I've added the padding back in.